ccd-definition/ComplexTypes/CaseManagementOrder.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CaseManagementOrder",
+    "ListElementCode": "hearingDate",
+    "FieldType": "Text",
+    "ElementLabel": "Which hearing is this order for?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CaseManagementOrder",
+    "ListElementCode": "hearingDateId",
+    "FieldType": "Text",
+    "ElementLabel": "Hearing Date Id",
+    "SecurityClassification": "Public",
+    "FieldShowCondition": "hearingDate=\"DO_NOT_SHOW\"",
+    "Comment": "Hearing date id has been used here to uniquely identify selected date in case management order"
+  }
```

Comment:
I think this is fine for now... but in future I would like to see a collection of caseManagementOrder where the id of the element is the hearingDateId. Will make retrieving from a list of elements much easier 👍